src/js/wall/LoginForm.tsx

```diff
@@ -41,13 +36,15 @@ export function LoginForm({ error }: LoginFormProps) {
         loginMutation.mutate(
             { username, password, remember },
             {
-                onSuccess: data => {
-                    dispatch({ type: LOGIN.SUCCEEDED, payload: data });
+                onSuccess: () => {
+                    dispatch(loginSucceeded());
```

Comment:
Good catch. I was unsure whether the payload was being utilized but you are right. I'll keep my old version instead of creating a new action since it is temporary and so that it doesn't get confused with the existing action.